src/ExpressionEvaluator/Core/Source/ExpressionCompiler/DkmUtilities.cs

```diff
@@ -43,19 +43,23 @@ private static IEnumerable<DkmClrModuleInstance> GetModulesInAppDomain(this DkmC
                 });
         }
 
-        internal unsafe static ImmutableArray<MetadataBlock> GetMetadataBlocks(this DkmClrRuntimeInstance runtime, DkmClrAppDomain appDomain)
+        internal static ImmutableArray<MetadataBlock> GetMetadataBlocks(
+            this DkmClrRuntimeInstance runtime, 
+            DkmClrAppDomain appDomain, 
+            ImmutableArray<MetadataBlock> previousMetadataBlocks = default(ImmutableArray<MetadataBlock>))
```

Comment:
Good point! Thank you!

---
In reply to: [127357938](https://github.com/dotnet/roslyn/pull/20834#discussion_r127357938) [](ancestors = 127357938)